homeassistant/components/vera/climate.py

```diff
@@ -40,10 +40,10 @@ async def async_setup_entry(
     async_add_entities: Callable[[List[Entity], bool], None],
 ) -> None:
     """Set up the sensor config entry."""
-    controller_data = hass.data[DOMAIN]
+    controller_data = get_controller_data(hass, entry.unique_id)
```

Comment:
```suggestion
    controller_data = get_controller_data(hass, entry.entry_id)
```

We usually store these by `entry.entry_id`